src/types/commands.ts

```diff
@@ -343,6 +343,7 @@ export interface DeleteSheetCommand extends SheetDependentCommand {
 export interface DuplicateSheetCommand extends SheetDependentCommand {
   type: "DUPLICATE_SHEET";
   sheetIdTo: UID;
+  sheetNameTo: string;
```

Comment:
should also have an allowDispatch to make sure we don't have a duplicate sheet name